### PR TITLE
Hello Branch is the branch of Hello Microverse's Project

### PR DESCRIPTION
**Pull request by engrabdullahriaz**
I have made Hello microverse project, and made changes like removed linter error, removed node_modules folder from project directory. sll checks were passes now.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (ShahierNashaat) [inline .gitignore:2]: - [x] Please add the node_modules to the .gitignore file.
- [x] Please remove the node_mocules folder from your project.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @engrabdullahriaz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Please fix the linters errors.
- [ ] Pull Request should have a descriptive title telling you what changes are inside.
- [ ] Pull Request should have a summary describing introduced changes in more detail.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (engrabdullahriaz) [reply .gitignore:2]: I have added node_modules to the .gitignore file and removed from project folder.

**Comment by engrabdullahriaz** [comment]
I have fixed linter file. removed node_modules folder from project directory. sll checks were passes now.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @engrabdullahriaz👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — COMMENTED** [review]
Hi @engrabdullahriaz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted an already approved pull request. Kindly open a new pull request with the changes made and request for a review with the new pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**